packages/lib/models/Setting.ts

```diff
@@ -613,6 +613,17 @@ class Setting extends BaseModel {
 				},
 			},
 
+			titleNaturalSort: {
+				value: false,
+				type: SettingItemType.Bool,
+				section: 'note',
+				public: true,
+				label: () => _('When sorting notes by title, use natural sorting'),
+				description: () => {
+					return _('Given notes with titles, e.g., A9 and A10, natural sorting orders A9 before A10');
+				},
+			},
+
```

Comment:
We won't add an option for this so either it's always on, or we don't add the feature. But from what I can see on the forum, most users would prefer natural sorting anyway. Is there any reason why it shouldn't always be on?

Reply:
I agree that it should always do natural sorting. The option has been removed.